### ISSUE TITLE
UTM crashes when I create and run a Windows 10 virtual machine on M1 

### ISSUE THREAD

**Issue by Salamit** [utmapp/UTM#3118]
**Describe the issue**

UTM crashes when I create and run a Windows 10 virtual machine on M1 

I followed the instructions here: https://www.youtube.com/watch?v=yWRzZYrSXDA. Upon pushing the play button for the first time, UTM hangs for a few minutes and then crashes.

**Configuration**
* UTM Version: 2.2.4 (36)
* OS Version: macOS 11.1 (20C69)
* Apple Silicon 

**Crash log**

[crashreport.pdf](https://github.com/utmapp/UTM/files/7177156/crashreport.pdf)

**Debug log**

[debug.pdf](https://github.com/utmapp/UTM/files/7177178/debug.pdf)

**Comment by conath** [comment]
From your crash log:

> OS Version:            macOS 11.1 (20C69)

Please update your Mac, this will resolve the issue.

Duplicate of #3098

**Comment by Salamit** [comment]
Noted and thanks!

**Comment by osy** [comment]
@conath even though updating macos fixes it I think we should force disable GL on < 11.3 to prevent crash.

**Comment by PJharii** [comment]
Same crash issue. Unfortunately I cannot update because i need IPA push support on my current mac os version. Any solution in my case?

**Comment by PJharii** [comment]
> From your crash log:
> 
> > OS Version:            macOS 11.1 (20C69)
> 
> Please update your Mac, this will resolve the issue.
> 
> Duplicate of #3098

Same crash issue. Unfortunately I cannot update because i need IPA push support on my current mac os version. Any solution in my case?

**Comment by osy** [comment]
@PJharii can you share your crash log as well?

**Comment by PJharii** [comment]
> @PJharii can you share your crash log as well?
[crash log.pdf](https://github.com/utmapp/UTM/files/7212229/crash.log.pdf)

**Comment by osy** [comment]
And debug log as well, sorry.

**Comment by PJharii** [comment]
> And debug log as well, sorry.

How to get it? Sorry.

**Comment by osy** [comment]
> For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Comment by PJharii** [comment]
> > For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.
[debug.log](https://github.com/utmapp/UTM/files/7212332/debug.log)

**Comment by osy** [comment]
Both @PJharii and @Salamit can you see if it works if you do not enable GL? In Graphics tab, choose the card that does NOT have -gl

**Comment by PJharii** [comment]
<img width="714" alt="Screenshot 2021-09-22 at 9 54 36 PM" src="https://user-images.githubusercontent.com/14293917/134383448-baa34fb5-a792-4289-b9c5-e4bb078d91f8.png">
Cannot find Graphics tab.

**Comment by osy** [comment]
Sorry, Display

**Comment by PJharii** [comment]
> Sorry, Display
<img width="785" alt="Screenshot 2021-09-22 at 9 59 03 PM" src="https://user-images.githubusercontent.com/14293917/134384106-b920adf7-f322-4a23-9408-77f88ef1af3b.png">
This is what I could see there

**Comment by osy** [comment]
Change Emulated Display Card to "virtio-ramfb" (no -gl)

**Comment by PJharii** [comment]
> Change Emulated Display Card to "virtio-ramfb" (no -gl)

dude, it is running now. No idea how far it will run. Also, one more query, i just went by the instructions given on official site. My macos harddisk and files wont be corrupted right?

**Comment by PJharii** [comment]
> Change Emulated Display Card to "virtio-ramfb" (no -gl)

Successfully booted. Thank you buddy. I couldn’t move the cursor out of that window. How can move the mouse out and in of the utm?

**Comment by conath** [comment]
Press ctrl & option/alt keys together to release the mouse.

**Comment by PJharii** [comment]
> Press ctrl & option/alt keys together to release the mouse.

Thank you. it worked. But I basically want windows on mac because I need to install a exe software. Can I do that on UTM. Or can I browse internet or connect usb pen drive?

**Comment by conath** [comment]
> […] I basically want windows on mac because I need to install a exe software. Can I do that on UTM. Or can I browse internet or connect usb pen drive?

Yes you can - however this is not the forum for discussing how to run VMs in UTM. Please see the [user guide](https://mac.getutm.app/guide/) and [GitHub discussions](https://github.com/utmapp/UTM/discussions). FWIW I will happily point you to my [YouTube guide](https://youtu.be/KsNoGPczavw) on running Windows in UTM.
